views/map.njk

```diff
@@ -0,0 +1,7 @@
+{% extends "./layouts/main.njk" %}
+
+{% block pageTitle %}
+  Map of Kainos Roles
```

Comment:
I think this is for another ticket

Reply:
That explains why after I fixed the issues on this branch and went back to my Map Ticket the file was empty! Corrected and committed.